jwst/datamodels/container.py

```diff
@@ -214,15 +215,9 @@ def from_asn(self, filepath, **kwargs):
             self.meta.asn_table._instance, asn_data
         )
 
-        # populate the output metadata with the output file from the ASN file
-        # Should remove the following lines eventually
-        self.meta.resample.output = str(asn_data['products'][0]['name'])
-        self.meta.table_name = str(filepath)
-        self.meta.pool_name = str(asn_data['asn_pool'])
-        self.meta.targname = str(asn_data['target'])
```

Comment:
I assume that the removal of targname, program, asn_type, and asn_rule is part of the effort to clean out rogue keyword definitions?

Reply:
Yes.  These ones were easy as they were not being used anywhere.  Will clean up the schemas in a separate PR.